testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -532,7 +548,7 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
 
     if create_db_per_test:
         log_info("Stopping the test server per test")
-        testserver.stop()
+        # testserver.stop()
```

Comment:
is this intentional?

Reply:
fixed, removed #, it should be uncommented